README.md

```diff
@@ -42,12 +42,17 @@ Dotenv.load ".env-other"
 # If you load env variable from file and
 # you want to raise execption in case of
 # missing dotenv file, to make this error
-# immediately obvious then use the bang 
+# immediately obvious then use the bang
 # verion of the load method:
 Dotenv.load!
 # or
 Dotenv.load! ".env-other"
 
+# Loading multiple files in order,
+# and sets the last loaded value for a varibale
+Dotenv.load! ".env" ".env.test"
```

Comment:
I think the example is not correct, the implementation is for array of strings.

```suggestion
Dotenv.load! %w(.env .env.test)
```

Reply:
Done!